src/tests/ftest/erasurecode/restart.py

```diff
@@ -18,7 +21,31 @@ class EcodServerRestart(ErasureCodeIor):
     def setUp(self):
         """Set up for test case."""
         super().setUp()
-        self.percent = self.params.get("size", '/run/ior/data_percentage/*')
+        self.percent = self.params.get("size", "/run/ior/data_percentage/*")
+        engine_count = self.server_managers[0].get_config_value("engines_per_host")
+        self.server_count = len(self.hostlist_servers) * engine_count
+
+    def verify_aggreation(self, expected_free_space):
+        """Verify aggregation by checking pool free space is greater than expected.
+
+        Args:
+            expected_free_space (int): expected free space after aggregation.
+        """
+        self.log.info("Waiting for aggregation to complete..")
+        if not wait_for_result(self.log, self.check_free_space, 150, delay=5,
+                               expected_free=expected_free_space):
+            self.fail("#aggregation completion not detected.")
```

Comment:
Recommend against putting `#` in failure messages - to keep it simple for Triage
```suggestion
            self.fail("aggregation completion not detected")
```